src/components/shareMenu/index.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { Input, Button, Typography, message } from 'antd';
+import TwitterIcon from '../../assets/images/twitter-icon.png';
+import EmailIcon from '../../assets/images/email-icon.png';
+import FBIcon from '../../assets/images/facebook-icon.png';
+import CopyIcon from '../../assets/images/copy-icon.png';
+
+const ShareMenuContainer = styled.div`
+  max-width: 550px;
+  margin: 20px 0px;
+`;
+
+const MediaShareButton = styled(Button)`
+  margin-top: 10px;
+  margin-right: 10px;
+  width: 60px;
+
+  &:hover {
+    background-color: #e7ffc7;
+  }
+`;
+
+const StyledImg = styled.img`
+  max-height: 25px;
+  margin-bottom: 3px;
+`;
+
+interface ShareMenuProps {
+  defaultText: string;
+  link: string;
+}
+
+const ShareMenu: React.FC<ShareMenuProps> = ({ defaultText, link }) => {
+  const [formText, setFormText] = useState(`${defaultText} ${link}`);
+
+  // goal is to open email window in new tab to stay consistent with more info and other share buttons
```

Comment:
Code is not a place for streams of consciousness. If it requires an explanation, refactor and add a purpose statment